lib/pymedphys/_dicom/connect/listen.py

```diff
@@ -115,9 +122,8 @@ def on_c_store(self, event):
         except KeyError:
             mode_prefix = "UN"
 
-        series_uid = dataset.SeriesInstanceUID
-        series_dir = self.storage_directory.joinpath(series_uid)
-        series_dir.mkdir(exist_ok=True)
+        series_dir = self._build_hierarchical_path_to_series(dataset)
+        os.makedirs(series_dir, exist_ok=True)
```

Comment:
Nit.

Potential to use pathlib directly for consistency?

```suggestion
        series_dir.mkdir(exist_ok=True, parents=True)
```